stream-chat-android-ui-components/src/main/res/values/attrs_message_list_view.xml

```diff
@@ -512,10 +512,26 @@
         <attr name="streamUiDisableScrollWhenShowingDialog" format="boolean" />
 
         <!-- The margin between the selected message and the edit reactions bubble on the options overlay. -->
-        <attr name="streamUiOptionsOverlayEditReactionsMargin" format="dimension|reference" />
+        <attr name="streamUiOptionsOverlayEditReactionsMarginBottom" format="dimension|reference" />
+        <!-- The margin between the edit reactions bubble on the options overlay and the parent. -->
+        <attr name="streamUiOptionsOverlayEditReactionsMarginTop" format="dimension|reference" />
+        <attr name="streamUiOptionsOverlayEditReactionsMarginStart" format="dimension|reference" />
```

Comment:
We are missing documentation for a few of these attributes, if you feel these are unimportant leave it as is.